index.html

```diff
@@ -26,6 +26,165 @@ <h1 class="career">I'm a software developer</h1>
+                    </section>
+                </div>
+            </section>
+            <section class="works-container">
+                <img class="image-grey" src="/Backgroud-images/Img_Plaholder.png" alt="image">
+                <div class="multi">
+                    <section class="multi-post">
+                        <h2 class="multi-post-title">Multi-Post Stories Gain+Glory</h2>
+                        <section class="buttons">
+                            <button class="languages">Ruby on rails</button>
+                            <button class="languages">css</button>
+                            <button class="languages">JavScript</button>
+                            <button class="languages">html</button>
+                        </section>
+                        <button class="see-project">See Project</button>
+                    </section>
+                </div>
+            </section>
+            <section class="works-container">
+                <img class="image-grey" src="/Backgroud-images/Img_Plaholder.png" alt="image">
+                <div class="multi">
+                    <section class="multi-post">
+                        <h2 class="multi-post-title">Multi-Post Stories Gain+Glory</h2>
+                        <section class="buttons">
+                            <button class="languages">Ruby on rails</button>
+                            <button class="languages">css</button>
+                            <button class="languages">JavScript</button>
+                            <button class="languages">html</button>
+                        </section>
+                        <button class="see-project">See Project</button>
+                    </section>
+                </div>
+            </section>
+            <section class="works-container">
+                <img class="image-grey" src="/Backgroud-images/Img_Plaholder.png" alt="image">
+                <div class="multi">
+                    <section class="multi-post">
+                        <h2 class="multi-post-title">Multi-Post Stories Gain+Glory</h2>
+                        <section class="buttons">
+                            <button class="languages">Ruby on rails</button>
+                            <button class="languages">css</button>
+                            <button class="languages">JavScript</button>
+                            <button class="languages">html</button>
+                        </section>
+                        <button class="see-project">See Project</button>
+                    </section>
+                </div>
+            </section>
+        </section>
+        <section class="whole-about-me-section">
+            <section class="about-me">
+                <div class="about-cont">
+                    <span></span>
+                    <h2 class="about-me-title">About me</h2>
+                    <img class="mask1" src="images/Mask_Group.png" alt="mask1">
```

Comment:
- [ ] OPTIONAL: I strongly suggest you to render this image as a background image from css. It will make a huge improvement on your page. Currently some items are not centred because you are trying to use `flexbox` to align the image and content. This is a wrong approach and will make more issues in. the future. Please consider rendering background images from `css` since they are not part of `html`